### ISSUE TITLE
Couldn't parse Facebook Problem

### ISSUE THREAD

**Issue by AlexeyDmitriev** [jmerle/competitive-companion#108]
https://www.facebook.com/codingcompetitions/hacker-cup/2020/qualification-round/problems/A
```
content.js:215 TypeError: Cannot read property 'querySelector' of null
    at common.js:5678
    at Array.map (<anonymous>)
    at FacebookCodingCompetitionsProblemParser.<anonymous> (common.js:5678)
    at Generator.next (<anonymous>)
    at common.js:5652
    at new Promise (<anonymous>)
    at push.__awaiter (common.js:5648)
    at FacebookCodingCompetitionsProblemParser.parse (common.js:5666)
    at content.js:211
    at Generator.next (<anonymous>)
(anonymous) @ content.js:215
```

I have version 2.17.2 which is last

**Comment by jmerle** [comment]
Also reported on [Codeforces](https://codeforces.com/blog/entry/60073?#comment-668692). I'll fix it early next week.

**Comment by AlexeyDmitriev** [comment]
Also, it looks like due to validation, input should be set back to regular expression

**Comment by jmerle** [comment]
The current parser should set the input filename to the name of the file you can download when submitting, although since I am on vacation without a laptop I can't check whether that is also the case in the qualification round.

**Comment by AlexeyDmitriev** [comment]
So, now you download to files, first you download file such as
travel_restrictions_validation_input.txt

and as long you submit correct solution for that you can download file such as
travel_restrictions_input.txt

**Comment by jmerle** [comment]
Oh I wasn't aware of validation input files, I'll switch it back to regex when I fix the parser.

**Comment by jmerle** [comment]
Fixed in [2.17.3](https://github.com/jmerle/competitive-companion/releases/tag/2.17.3) which has just gone live. It has already been approved on Mozilla's AMO but is still pending approval in the Chrome Web Store, so depending on which browser you use it may take a few days until you receive the update.
